src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
-    public String[] readFromFile(String fileName) {
-        return null;
+    public String[] readFromFile(String fileName) throws IOException {
+        File file = new File(fileName);
+        String result = "";
+
+        try {
+            BufferedReader bufReader = new BufferedReader(new FileReader(file));
+            StringBuilder strBuilder = new StringBuilder();
+            String value = bufReader.readLine();
+
+            while (value != null) {
+                String[] words = value.split(" ");
```

Comment:
let's make dividers, regex and character we are looking for constant variables

Reply:
done